wandb/sdk/wandb_settings.py

```diff
@@ -384,9 +485,9 @@ def _path_convert(self, *path):
         path = os.path.expanduser(path)
         return path
 
-    def _clear_early_logger(self):
-        # TODO(jhr): this is a hack
-        object.__setattr__(self, "_early_logger", None)
+    # def _clear_early_logger(self):
```

Comment:
Do we still need this?

Reply:
not being used right now, but I left it in as a note to consider.  WIll remove after I convert all users to new settings rules